### PR TITLE
add tools menu and related panel

### PR DESCRIPTION
**Pull request by gordonleacock**
Print tool doesn't display yet
Citation tools doesn't work yet
Related entries shows title and panel but not content yet

Otherwise, everything's great! ;-)

### CODE REVIEW

File: app/views/catalog/_print.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<%# file _print.html.erb %>
+<% @documents.entry.each do |document| %>
+
+  <h3 class="modal-title"><%= document_heading(document) %></h3>
+
+  <%# PUT document.send(:print) here; example from citation below %>
+
+       <% if document.respond_to?(:export_as_mla_citation_txt) %>
+         <h4><%#= t('blacklight.citation.mla') %></h4>
+         <%3= document.send(:export_as_mla_citation_txt).html_safe %><br/><br/>
```
Comment (billdueber) [inline app/views/catalog/_print.html.erb]: What does the '3' here do? (or is it just a typo?)

File: app/views/catalog/_print.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+       <% end %>
+
+       <% if document.respond_to?(:export_as_apa_citation_txt) %>
+         <h4><%#= t('blacklight.citation.apa') %></h4>
+         <%#= document.send(:export_as_apa_citation_txt).html_safe %><br/><br/>
```
Comment (billdueber) [inline app/views/catalog/_print.html.erb]: Also, in the interest of at least pretending we're doing some code review: this code seems overly defensive to me. The list of "responds to" and then using `#send(:methodname)` feels like it's designed for a loop (which we're not using) and assumes that the representable objects don't have a unified interface (which they darn well should). Let's think about a cleaner way to do this once we get this damn thing out the door.

Comment (gordonleacock) [reply app/views/catalog/_print.html.erb]: Okay. I'm following the citation's blacklight pattern here, which may be overly defensive, especially for a print function.

File: app/views/catalog/_print.html.erb (hunk 1 above)
Comment (gordonleacock) [reply app/views/catalog/_print.html.erb]: Yes, it is a typo!

File: app/views/catalog/_print.html.erb (hunk 2 above)
Comment (gordonleacock) [reply app/views/catalog/_print.html.erb]: And I removed the "responds to" ifs.

**Pull request merged**